include/linux/blkdev_compat.h

```diff
@@ -351,8 +351,8 @@ bio_set_flush(struct bio *bio)
 	bio_set_op_attrs(bio, 0, REQ_PREFLUSH);
 #else
 #error	"Allowing the build will cause bio_set_flush requests to be ignored."
-	"Please file an issue report at: "
-	"https://github.com/zfsonlinux/zfs/issues/new"
+#error	"Please file an issue report at: "
+#error	"https://github.com/zfsonlinux/zfs/issues/new"
```

Comment:
Let's go with a single line, and for the last instance use.  The additional information can simply be moved in to a comment.  The goal of these checks is primarily to prevent a potentially unsafe build on a non-tested kernel.

```
#error  "Allowing the build will cause discard requests to become writes."
```